tests/test_plugin_binance_us.py

```diff
@@ -0,0 +1,893 @@
+            api_key="a",
+            api_secret="b",
+            native_fiat="USD",
+        )
+
+        client = plugin._client
+
+        mocker.patch.object(client, "fetch_markets").return_value = [{"id": "ETHBTC"}]
+        mocker.patch.object(plugin, "_AbstractCcxtInputPlugin__start_time_ms", int(datetime.datetime.now().timestamp()) * 1000 - 1)
+        mocker.patch.object(client, "fetch_withdrawals").return_value = [
+            {
+                "info": {
+                    "amount": "0.00999800",
+                    "coin": "PAXG",
+                    "network": "ETH",
+                    "status": "1",
+                    "address": "0x788cabe9236ce061e5a892e1a59395a81fc8d62c",
+                    "addressTag": "",
+                    "txId": "0xaad4654a3234aa6118af9b4b335f5ae81c360b2394721c019b5d1e75328b09f3",
+                    "insertTime": "1599621997000",
+                    "transferType": "0",
+                    "confirmTimes": "12/12",
+                    "unlockConfirm": "12/12",
+                    "walletType": "0",
+                },
+                "id": None,
+                "txid": "0xaad4654a3234aa6118af9b4b335f5ae81c360b2394721c019b5d1e75328b09f3",
+                "timestamp": 1599621997000,
+                "datetime": "2020-09-09T03:26:37.000Z",
+                "network": "ETH",
+                "address": "0x788cabe9236ce061e5a892e1a59395a81fc8d62c",
+                "addressTo": "0x788cabe9236ce061e5a892e1a59395a81fc8d62c",
+                "addressFrom": None,
+                "tag": None,
+                "tagTo": None,
+                "tagFrom": None,
+                "type": "withdrawal",
+                "amount": 0.00999800,
+                "currency": "PAXG",
+                "status": "ok",
+                "updated": None,
+                "internal": False,
+                "fee": None,
+            }
+        ]
+
+        mocker.patch.object(plugin, "_process_trades").return_value = None
+        #mocker.patch.object(plugin, "_process_gains").return_value = None
+        mocker.patch.object(plugin, "_process_deposits").return_value = None
+        #mocker.patch.object(plugin, "_process_implicit_api").return_value = None
+
+        result = plugin.load(US())
+
+        # 1 crypto Transfer
+        assert len(result) == 1
+
+        crypto_withdrawal_transaction: IntraTransaction = result[0]  # type: ignore
+
+        assert crypto_withdrawal_transaction.asset == "PAXG"
+        assert int(parser.parse(crypto_withdrawal_transaction.timestamp).timestamp()) * 1000 == 1599621997000
+        assert crypto_withdrawal_transaction.to_exchange == Keyword.UNKNOWN.value
+        assert crypto_withdrawal_transaction.from_exchange == "Binance.us"
+        assert crypto_withdrawal_transaction.crypto_received == Keyword.UNKNOWN.value
+        assert RP2Decimal(crypto_withdrawal_transaction.crypto_sent) == RP2Decimal("0.00999800")
+
+
+"""
+    def test_gains(self, mocker: Any) -> None:
```

Comment:
if you want to disable a test, you can simply put disabled in front of the method name:
```suggestion
    def disabled_test_gains(self, mocker: Any) -> None:
```

Reply:
Of course, if you never need gains, you can just delete the test entirely. You can always copy and paste the code from binance in the future if you think you need it.